packages/snaps-controllers/src/services/AbstractExecutionService.ts

```diff
@@ -397,26 +397,6 @@ export abstract class AbstractExecutionService<WorkerType>
     this.#snapRpcHooks.set(snapId, rpcHook);
   }
 
-  /**
-   * Gets the job id for a given snap.
-   *
-   * @param snapId - A given snap id.
-   * @returns The ID of the snap's job.
-   */
-  #getJobForSnap(snapId: string): string | undefined {
-    return this.#snapToJobMap.get(snapId);
-  }
-
-  /**
-   * Gets the snap id for a given job.
-   *
-   * @param jobId - A given job id.
-   * @returns The ID of the snap that is running the job.
-   */
-  #getSnapForJob(jobId: string): string | undefined {
-    return this.#jobToSnapMap.get(jobId);
-  }
```

Comment:
These functions were unused. Since they're hash private it's safe to remove them.